client/app/telecom/pack/xdsl/modem/connectedDevices/pack-xdsl-modem-connectedDevices.html

```diff
@@ -19,26 +19,27 @@
                     data-translate-attr="{ title : 'xdsl_modem_device_connected' }"
                     type="button"
                     data-ng-click="DeviceCtrl.showDevices = !DeviceCtrl.showDevices">
-                <i class="ovh-font"
-                      data-ng-class="{
-                          'ovh-font-disclosure-right':!DeviceCtrl.showDevices,
-                          'ovh-font-disclosure-down':DeviceCtrl.showDevices}"
-                      aria-hidden="true">
-                </i>
-                <span data-ng-pluralize
-                      data-ng-if="DeviceCtrl.devices"
-                      data-count="DeviceCtrl.devices.length"
-                      data-when="{
-                          '0': ('xdsl_modem_device_show_none' | translate),
-                          'one': ('xdsl_modem_device_show_one' | translate : {count: DeviceCtrl.devices.length}),
-                          'other': ('xdsl_modem_device_show_many' | translate : {count: DeviceCtrl.devices.length})}">
+                <span data-ng-if="DeviceCtrl.devices.length">
+                    <i class="ovh-font"
+                          data-ng-class="{
+                              'ovh-font-disclosure-right':!DeviceCtrl.showDevices,
+                              'ovh-font-disclosure-down':DeviceCtrl.showDevices}"
+                          aria-hidden="true">
+                    </i>
+                    <span data-ng-pluralize
+                          data-count="DeviceCtrl.devices.length"
+                          data-when="{
+                              '0': ('xdsl_modem_device_show_none' | translate),
```

Comment:
The `when` will never go through the `0` value because of  `data-ng-if="DeviceCtrl.devices.length"` (if the length is equal to 0, the `ngIf` will be evaluated to `false`)